python/cugraph/cugraph/tests/test_triangle_count.py

```diff
@@ -114,6 +114,51 @@ def test_triangles_no_start(input_combo):
             assert len(counts_diff) == 0
 
 
+def test_triangle_int64(input_combo):
+    G_32 = input_combo["G"]
+    count_legacy_32 = cugraph.triangles(G_32)
+
+    graph_files = input_combo["graph_file"]
+    gdf = cudf.read_csv(graph_files,
+                        delimiter=' ',
+                        dtype=['int64', 'int64', 'float32'],
+                        header=None)
+    G = cugraph.Graph()
+    G.from_cudf_edgelist(gdf, source='0', destination='1', edge_attr='2')
+
+    count_exp_64 = cugraph.experimental.triangle_count(G).sort_values(
+                "vertex").reset_index(drop=True).rename(columns={
+                    "counts": "exp_cugraph_counts"})
+    cugraph_exp_triangle_results = \
+        count_exp_64["exp_cugraph_counts"].sum()
+    assert G.edgelist.edgelist_df['src'].dtype == 'int64'
+    assert G.edgelist.edgelist_df['dst'].dtype == 'int64'
+    assert cugraph_exp_triangle_results == count_legacy_32
```

Comment:
Can you rename `cugraph_exp_triangle_results` to `count_legacy_64` for consistency?